lib/senkyoshi/models/questions/matching.rb

```diff
@@ -9,6 +9,18 @@ def initialize
       @distractors = []
     end
 
+    def strip_select_html(text)
+      tag_patterns = [/<\/?p[^>]*>/i,             # <p> tags
```

Comment:
also, if you do it as a hash you can define what each pattern should be replaced with. Not that we need that exactly here, but just for your knowledge. :]